### PR TITLE
Removed proprietary mentions from file headers

### PR DESCRIPTION
**Pull request by dvrogozh**
Fixes: #2937

Signed-off-by: Dmitry Rogozhkin <dmitry.v.rogozhkin@intel.com>

### CODE REVIEW

File: tools/tracer/gui/Program.cs [hunk 1]
```diff
@@ -1,14 +1,22 @@
...
-INTEL CORPORATION PROPRIETARY INFORMATION
-This software is supplied under the terms of a license agreement or nondisclosure
-agreement with Intel Corporation and may not be copied or disclosed except in
-accordance with the terms of that agreement
-Copyright(c) 2012-2020 Intel Corporation. All Rights Reserved.
-
-File Name: Program.cs
-
-\* ****************************************************************************** */
+// Copyright (c) 2015-2022 Intel Corporation
```
Comment (lwan108) [inline tools/tracer/gui/Program.cs:1]: Should data be aligned with previous one, like 2015-2022?

File: tools/tracer/gui/SdkAnalyzerForm.cs [hunk 2]
```diff
@@ -1,14 +1,22 @@
...
-INTEL CORPORATION PROPRIETARY INFORMATION
-This software is supplied under the terms of a license agreement or nondisclosure
-agreement with Intel Corporation and may not be copied or disclosed except in
-accordance with the terms of that agreement
-Copyright(c) 2020 Intel Corporation. All Rights Reserved.
-
-File Name: SdkAnalyzer.cs
-
-\* ****************************************************************************** */
+// Copyright (c) 2015-2022 Intel Corporation
```
Comment (lwan108) [inline tools/tracer/gui/SdkAnalyzerForm.cs:1]: 2020-2022

**Review by lwan108 — CHANGES_REQUESTED** [review]

File: tools/tracer/gui/Program.cs (hunk 1 above)
Comment (dvrogozh) [reply tools/tracer/gui/Program.cs]: done

File: tools/tracer/gui/SdkAnalyzerForm.cs (hunk 2 above)
Comment (dvrogozh) [reply tools/tracer/gui/SdkAnalyzerForm.cs]: done

**Review by lwan108 — APPROVED** [review]
LGTM

**Comment by tjyrinki** [comment]
Thank you!

**Pull request merged**